### PR TITLE
Add samples for handling GDPR compliance

### PR DESCRIPTION
**Pull request by Mriegel**

### CODE REVIEW

File: SDK-v5/gdpr-places/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# GDPR Compliant example
+
+Sample project demonstrating the integration of Localytics for a customer supporting GDPR. This app ensures that the correct opted out status is always reported to Localytics, including when that status has changed in another service, such as a webpage.
```
Comment (kbataran) [inline SDK-v5/gdpr-places/README.md]: Can you add a link to the Advanced>Privacy section of the docs here?

File: SDK-v5/gdpr-places/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+This app requires SDK v5.1 and up in order to properly support GDPR. 
+
+## Setup
+
+1. In `localytics.xml` replace `YOUR-LOCALYTICS-APP-KEY` with your Localytics app key.
+2. Replace empty sections left in `UserLoginTask#doInBackground` and `MyApplication` with appropriate server calls to log a user in as well as check their privacy opted out status. 
+3. Build and run.
+
+## Summary
+This app is gated by an authentication screen.  It is required that the end user log into the app in order to use the underlying features.  As a result, we don't deal with anonymous users, and can always garauntee a GDPR compliant integration for end users. This app additionally re-enables Places so that location based messaging can still target the end user.
```
Comment (kbataran) [inline SDK-v5/gdpr-places/README.md:18]: end user or end-user?

File: SDK-v5/gdpr-places/app/src/main/java/com/localytics/gdpr/LoginActivity.java [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    private boolean isEmailValid(String email) {
+        return email.contains("@");
+    }
+
+    private boolean isPasswordValid(String password) {
+        return password.length() > 4;
+    }
+
+    /**
+     * Shows the progress UI and hides the login form.
```
Comment (kbataran) [inline SDK-v5/gdpr-places/app/src/main/java/com/localytics/gdpr/LoginActivity.java:226]: You really went all out with this, love it

File: SDK-v5/gdpr-places/app/src/main/java/com/localytics/gdpr/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        Localytics.setCustomerIdWithPrivacyOptedOut(null, false);
+        SharedPreferencesUtil.setUserLoggedIn(this, false);
+        navigateToLogin();
+    }
+
+    private void setOptOutStatus(final boolean privacyOptedOut) {
+        Localytics.setPrivacyOptedOut(privacyOptedOut);
+        Localytics.pauseDataUploading(false);
+        Localytics.setLocationMonitoringEnabled(true);
+        //Make a call to your servers to update the status of data collection opt out for this user.
```
Comment (kbataran) [inline SDK-v5/gdpr-places/app/src/main/java/com/localytics/gdpr/MainActivity.java]: i'd add "TODO: " for clarity

File: SDK-v5/gdpr-places/app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/colorPrimary</item>
```
Comment (kbataran) [inline SDK-v5/gdpr-places/app/src/main/res/values/styles.xml:6]: should've used blanched almond...

**Review by kbataran — APPROVED** [review]
Approving, all comments are simple edits

File: SDK-v5/gdpr-places/README.md (hunk 2 above)
Comment (Mriegel) [reply SDK-v5/gdpr-places/README.md:18]: opted for end user on the docs so this should be good.

**Comment by kbataran** [comment]
Looks great!

**Pull request merged**